db/schema.rb

```diff
@@ -382,6 +382,7 @@
 
   create_table "learning_hours", force: :cascade do |t|
     t.bigint "user_id", null: false
+    t.integer "learning_type", default: 5
```

Comment:
Should this be in here?

Reply:
It's from merging main into the branch. Can back it out. Looks like there won't be merge conflicts.